extensions/netty/runtime/src/main/java/io/quarkus/netty/runtime/graal/HttpContentCompressorSubstitutions.java

```diff
@@ -37,7 +37,9 @@ public IsZstdAbsent() {
             try {
                 Class.forName("com.github.luben.zstd.Zstd");
                 zstdAbsent = false;
-            } catch (ClassNotFoundException e) {
+            } catch (Exception e) {
```

Comment:
Should we catch `Error`s too, just in case?

```suggestion
            } catch (Throwable e) {
```